tests/sedov-blast/sedov-blast-parameter-variation.sh

```diff
@@ -2,11 +2,11 @@ NUM_PARALLEL_PROCESSORS=8
 testNames=(fom online)
 case $subTestNum in
   1)
-    $LAGHOS -m data/cube01_hex.mesh -pt 211 -tf 0.01 -offline -romsvds -romos -romsrhs -bef 1.0 -rpar 0
+    $LAGHOS -m data/cube01_hex.mesh -pt 211 -tf 0.01 -offline -romsvds -romos -rostype previous -romsrhs -bef 1.0 -rpar 0
```

Comment:
If the default value of `-rostype` is `previous`, this does not need to be specified, right? Anyway, we can keep it like this for clarity.

Reply:
Yes they are not needed in this branch and any future branches. They are added to make sure the regression tests results match with the current rom-dev branch, which use `load` as default, as well as make it more clear.